spec.html

```diff
@@ -8258,7 +8258,6 @@
 
       <emu-clause id="sec-set-immutable-prototype">
         <h1>SetImmutablePrototype ( _O_, _V_ )</h1>
-        <!-- This is used by host environment specs, even though it only has one consumer in this spec; don't inline it -->
```

Comment:
How is this stale? I don't see anywhere else in the spec that references it (unless somewhere else in this PR references it?)

Either way, it seems prudent to leave it in, in case the other uses later get factored out.